cmd/node/factory/structsStatusHandlers.go

```diff
@@ -43,12 +47,18 @@ func NewStatusHandlersFactoryArgs(
 	ctx *cli.Context,
 	marshalizer marshal.Marshalizer,
 	uint64ByteSliceConverter typeConverters.Uint64ByteSliceConverter,
+	chanStartViews chan bool,
+	chanLogRewrite chan bool,
+	logFile *os.File,
 ) *ArgStatusHandlers {
 	return &ArgStatusHandlers{
 		LogViewName:              logViewName,
 		Ctx:                      ctx,
 		Marshalizer:              marshalizer,
 		Uint64ByteSliceConverter: uint64ByteSliceConverter,
+		ChanStartViews:           chanStartViews,
```

Comment:
both channels are not checked for nil, code might panic.

Reply:
added checks for all parameters